pegen/pegen.c

```diff
@@ -613,7 +613,7 @@ pegen_alias(alias_ty alias, int line, int endline, int col, int endcol, PyArena
     a->lineno = line;
     a->end_lineno = endline;
     a->col_offset = col;
```

Comment:
I'd swap the above two lines, just to keep things in a consistent order with the arguments. Maybe also make sure that the fields are declared in the same order -- it should always be `lineno, col_offset, end_lineno, end_col_offset`. Making the names consistent everywhere also might be a good idea -- just so there's less need to think. :-)